src/core/components/accordion/accordionItemHeader/accordionItemHeader.tsx

```diff
@@ -0,0 +1,31 @@
+import {
+    AccordionHeader as RadixAccordionHeader,
+    AccordionTrigger as RadixAccordionTrigger,
+} from '@radix-ui/react-accordion';
+import classNames from 'classnames';
+import { forwardRef, type ComponentPropsWithRef } from 'react';
+import { AvatarIcon } from '../../avatars';
+import { IconType } from '../../icon';
+
```

Comment:
Let's add an `IAccordionItemHeaderProps` interface here and all the other components that we can expand later if needed. We can use it on the createTesetComponent function. Make sure to export it from the /accordionItemHeader/index.ts` file as well